core/src/main/scala/org/apache/spark/broadcast/BroadcastManager.scala

```diff
@@ -52,8 +52,8 @@ private[spark] class BroadcastManager(
 
   private val nextBroadcastId = new AtomicLong(0)
 
-  def newBroadcast[T: ClassTag](value_ : T, isLocal: Boolean): Broadcast[T] = {
-    broadcastFactory.newBroadcast[T](value_, isLocal, nextBroadcastId.getAndIncrement())
+  def newBroadcast[T: ClassTag](_value: T, isLocal: Boolean): Broadcast[T] = {
```

Comment:
I am not sure of this naming change. I mean the change does not look particulaily better to me. I believe we have a open PR - https://github.com/databricks/scala-style-guide/pull/52 which appearntly stays against some cases here.